packages/cosmic-swingset/test/make.test.js

```diff
@@ -218,9 +218,203 @@ const walletProvisioning = test.macro({
       retryCount < retryCountMax,
       `wallet is provisioned within ${retryCount} retries out of ${retryCountMax}`,
     );
-
     await verifier?.stop(t);
   },
 });
 
 test.serial(walletProvisioning, 'wallet provisioning');
+
+const prometheusUrl = 'http://localhost:26660/metrics';
+const testMetricNames = [
+  'store_size_decrease{storeKey="vstorage"}',
+  'store_size_increase{storeKey="vstorage"}',
+];
+const testMetricNamesSet = new Set(testMetricNames);
+
+/**
+ * getMetrics fetches the metrics from the Prometheus URL and returns the metrics
+ * as an object with metric names as keys and their values as values.
+ *
+ * @param {*} url             Prometheus URL
+ * @param {*} metricNames     Array of metric names to fetch
+ * @returns {Promise<Record<string, any>>}
```

Comment:
It's relevant that the object returned by this function only includes the selected metrics.
```suggestion
 * getMetrics reads data from the Prometheus URL and returns the selected
 * metrics as an object.
 *
 * @param {string} url            Prometheus URL
 * @param {string[]} metricNames  Selected metric names
 * @returns {Promise<Record<string, number>>}
```

Reply:
Yes, agree.